Rear Rider/Rear Rider/HomeView.swift

```diff
@@ -26,8 +27,14 @@ struct HomeView: View {
                     }
                 }
 
-                Image(systemName: "wifi.circle")
-                    .foregroundColor(.green)
+                if wifiManager.wifiOn {
+                    Image(systemName: "wifi.circle")
+                        .foregroundColor(.green)
+                }
+                else {
+                    Image(systemName: "wifi.circle")
+                        .foregroundColor(.red)
+                }
```

Comment:
```suggestion
Image(systemName: "wifi.circle")
    .foregroundColor(wifiManager.wifiOn ? .green : .red)
```

Non-blocking, but if you're only changing one little thing a ternary operator is cleaner than a big if/else

Reply:
Yes that looks better. I could do it in the next branch for the Bluetooth icon too